server-code/Reports/Reports.ts

```diff
@@ -92,6 +92,7 @@ function getOpenChangeRequests() {
 * @return {String} - Constructed HTML table listing all the fab and weld reports
 */
 function getAllFabLogs() {
-	var data = getSheetInfo(MAIN_SHEET_ID_STR, FAB_WELD_STR, DATA_STR);
+    var data = getSheetInfo(MAIN_SHEET_ID_STR, FAB_WELD_STR, DATA_STR);
+    transformToHyperLinks(projectList, ["Links"]);
```

Comment:
Please wait for James or I to merge the PR next time, thanks. There seems to be an error in your code. I think you meant to put 'data' here instead of 'projectList'.